docs/setup/config.md

```diff
@@ -389,6 +389,10 @@ These parameters allow for advanced tuning. The default values are sufficient wh
 
 - `jobmanager.web.access-control-allow-origin`: Enable custom access control parameter for allow origin header, default is `*`.
 
+### Web Frontend
```

Comment:
True but this is the `JobManager Web Frontend`. In the future (with Flip-6) there will be more than one web frontend component which is not necessarily tied to the `JobManager`. Therefore, I though about introducing some general web frontend configuration options. What do you think?